includes/helper.php

```diff
@@ -122,6 +117,16 @@ function auto_populate_fields_action_tag_semaphore($misc, $action_tag, $return_v
     return false;
 }
 
+function get_action_tags() {
```

Comment:
Let's keep the pattern `auto_populate_fields_` in order to avoid generic names that may cause conflicts with core and/or future modules.

Let's also add a description header for this function.